eclair-core/src/main/scala/fr/acinq/eclair/payment/Bolt11Invoice.scala

```diff
@@ -167,7 +167,7 @@ object Bolt11Invoice {
         fallbackAddress.map(FallbackAddress(_)),
         expirySeconds.map(Expiry(_)),
         Some(MinFinalCltvExpiry(minFinalCltvExpiryDelta.toInt)),
-        Some(InvoiceFeatures(features))
+        Some(InvoiceFeatures(features.copy(unknown = Set.empty).unscoped()))
```

Comment:
Can you add a comment here?
```suggestion
        // We want to keep invoices as small as possible, so we explicitly remove unknown features.
        Some(InvoiceFeatures(features.copy(unknown = Set.empty).unscoped()))
```

Reply:
Done